pkg/kafka/marshaler.go

```diff
@@ -66,6 +72,15 @@ func (DefaultMarshaler) Unmarshal(kafkaMsg *sarama.ConsumerMessage) (*message.Me
 	return msg, nil
 }
 
+func (DefaultMarshaler) UnmarshalWithContext(ctx context.Context, kafkaMsg *sarama.ConsumerMessage) (*message.Message, error) {
+	msg, err := DefaultMarshaler{}.Unmarshal(kafkaMsg)
```

Comment:
```suggestion
func (m DefaultMarshaler) UnmarshalWithContext(ctx context.Context, kafkaMsg *sarama.ConsumerMessage) (*message.Message, error) {
	msg, err := m.Unmarshal(kafkaMsg)
```
Would that make sense?

Reply:
I think that makes sense!